metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -282,6 +287,29 @@ def _register_job_definition(
                     {"sourceVolume": name, "containerPath": host_path}
                 )
 
+        if use_tmpfs or tmpfs_size:
```

Comment:
maybe we could drop `or tmpfs_size` here.  `@batch(use_tmpfs=False, tmpfs_size=10000)` leads to this error:
```
 FileNotFoundError: [Errno 2] No such file or directory: '/metaflow_temp/metaflow.s3.evqtjhbe'
 ```

we can interpret that `use_tmpfs=False` means disabling all `tmpfs` behavior altogether

Reply:
(this error is probably related to the other comment related to `if tmpfs_tempdir` that I made below)